### ISSUE TITLE
Error parsing args when use_papa = True

### ISSUE THREAD

**Issue by cgarciaarano** [circus-tent/circus#930]
Hi, 

we've found that Circus 0.12.2 is not parsing args correctly when `use_papa = True`. With this config file:

```ini
[circus]
check_delay = 5
endpoint = tcp://127.0.0.1:5555
pubsub_endpoint = tcp://127.0.0.1:5556
statsd = true

[watcher:app1]
cmd = ping
args = 8.8.8.8
use_papa=True
```
The debug log shows:

```log
2015-09-16 19:35:57 circus[17841] [DEBUG] cmd: ping
2015-09-16 19:35:57 circus[17841] [DEBUG] args: ['8', '.', '8', '.', '8', '.', '8']
2015-09-16 19:35:57 circus[17841] [DEBUG] process args: ['ping', '8', '.', '8', '.', '8', '.', '8']
```
and the process keeps flapping.

Commenting out in `papa_process_proxy.py` the `self.args` assign:
```python
    def spawn(self):
        # noinspection PyUnusedLocal
        socket_names = set(socket_name.lower()
                           for socket_name in self.watcher._get_sockets_fds())
        self.cmd = self._fix_socket_name(self.cmd, socket_names)
        #self.args = [self._fix_socket_name(arg, socket_names)
        #             for arg in self.args]
        args = self.format_args()
        stdout = _bools_to_papa_out(self.pipe_stdout, self.close_child_stdout)
        stderr = _bools_to_papa_out(self.pipe_stderr, self.close_child_stderr)
```

seems to fix this behaviour, because args is being processed as a string, not a list. I'm not really sure what `_fix_socket_name()` is doing. Maybe this fix is enough?

```python
self.args = [self._fix_socket_name(arg, socket_names)
                     for arg in self.args.split(' ')]
```

Regards

**Comment by ayashjorden** [comment]
Hi, 
I'm having the same issue.
Any time frame for the fix?

Thank you,
Yarden

**Comment by k4nar** [comment]
@scottkmaxwell : Can I assign you to this one ?

**Comment by cgarciaarano** [comment]
I can pull request my proposal, but I'm not really sure it's correct, it could have side effects.

**Comment by k4nar** [comment]
We are already splitting the args at some point, so I think that `self._fix_socket_name` should be applied at this point.

**Comment by scottkmaxwell** [comment]
Yeah, it sounds like I was expecting args to already be a list at this point. I am completely swamped at the moment but I can try to have a look toward the end of the week if somebody hasn't figured out a proper fix yet.

**Comment by ayashjorden** [comment]
@scottkmaxwell any update here?

**Comment by scottkmaxwell** [comment]
Sorry this took so long.

**Comment by ayashjorden** [comment]
@k4nar when can we expect a release containing this fix?

**Comment by k4nar** [comment]
@ayashjorden : Sorry for the delay, I'll try to make a new release this week.

**Comment by ayashjorden** [comment]
Thank you @k4nar :)

**Comment by antgel** [comment]
Hey all, has the fix been released yet?

**Comment by k4nar** [comment]
Sorry, I've been experiencing some troubles with our tests suite (cf #984 for progress), and I would really like to fix that before releasing a new version. I'll push as much effort as I can to make it happen this week.

**Comment by k4nar** [comment]
I've merged #984. I'm giving some time to the opened PRs to be rebased & merged, and I'll release a new version.

**Comment by k4nar** [comment]
@ayashjorden : I just released Circus [0.14](https://github.com/circus-tent/circus/releases/tag/0.14.0). Sorry for the (very) long delay!

**Comment by ayashjorden** [comment]
Thank you! @k4nar
